modules/hitl2/src/views/full/Components/CommentForm.tsx

```diff
@@ -9,23 +13,22 @@ const CommentForm: FC<Props> = props => {
   const [content, setContent] = useState('')
 
   return (
-    <div style={{ display: 'flex' }}>
-      <TextArea
-        fill={true}
+    <div className={style.commentForm}>
+      <textarea
         value={content}
         onChange={event => {
           setContent(event.target.value)
         }}
-      ></TextArea>
+      ></textarea>
       <Button
+        icon={<SendIcon />}
         onClick={() => {
+          // tslint:disable-next-line: no-floating-promises
```

Comment:
Why that tslint comment, isn't it not floating ?

Reply:
Just muting the linting as it was showing up as a no-floating-promises, not my code so I'm not too sure